Description: Fix parallel build failure.
 Reverts 6abc41e68c5385f97cf9dea25b0e87cd9382c92d
Author: Bas Couwenberg <sebastic@debian.org>
Bug: https://github.com/OSGeo/gdal/issues/707

--- a/swig/perl/GNUmakefile
+++ b/swig/perl/GNUmakefile
@@ -29,7 +29,7 @@ build: gdal_wrap.cc Makefile_Geo__GDAL
 
 gdal_wrap.cc: generate
 
-Makefile_Geo__GDAL: gdal_wrap.cc
+Makefile_Geo__GDAL:
 	perl Makefile.PL INSTALLDIRS=vendor
 
 test: build
